### PR TITLE
Update vendoring

### PR DESCRIPTION
**Pull request by SuperQ**
* github.com/Sirupsen/logrus
* github.com/beorn7/perks/quantile
* github.com/go-sql-driver/mysql
* github.com/golang/protobuf/proto
* github.com/gopherjs/gopherjs/js
* github.com/jtolds/gls
* github.com/matttproud/golang_protobuf_extensions/pbutil
* github.com/prometheus/client_golang/prometheus
* github.com/prometheus/client_model/go
* github.com/prometheus/common/expfmt
* github.com/prometheus/common/internal/bitbucket.org/ww/goautoneg
* github.com/prometheus/common/log
* github.com/prometheus/common/model
* github.com/prometheus/common/version
* github.com/prometheus/procfs
* github.com/smartystreets/assertions
* github.com/smartystreets/assertions/internal/go-render/render
* github.com/smartystreets/assertions/internal/oglematchers
* github.com/smartystreets/goconvey/convey
* github.com/smartystreets/goconvey/convey/gotest
* github.com/smartystreets/goconvey/convey/reporting
* golang.org/x/sys/unix
* golang.org/x/sys/windows
* golang.org/x/sys/windows/registry
* golang.org/x/sys/windows/svc/eventlog
* gopkg.in/DATA-DOG/go-sqlmock.v1
* gopkg.in/ini.v1

---
This change is [Reviewable](https://reviewable.io/reviews/prometheus/mysqld_exporter/168)

### CODE REVIEW

**Comment by SuperQ** [comment]
@beorn7 Ping, this updates the client_golang for https://github.com/prometheus/mysqld_exporter/issues/127

File: vendor/vendor.json [hunk 1]
```diff
@@ -29,101 +43,130 @@
...
-			"revisionTime": "2015-10-11T12:25:29+02:00"
+			"revision": "c12348ce28de40eed0136aa2b644d0ee0650e56c",
+			"revisionTime": "2016-04-24T11:30:07Z"
 		},
 		{
+			"checksumSHA1": "GGb6UcrDCNMWWIjyjDgvauGw9qI=",
 			"path": "github.com/prometheus/client_golang/prometheus",
-			"revision": "18acf9993a863f4c4b40612e19cdd243e7c86831",
-			"revisionTime": "2016-02-23T13:27:30+01:00"
+			"revision": "e56081f7b9d0c522b3b3b3ef154be6d74001382f",
```
Comment (beorn7) [inline vendor/vendor.json]: For consistency, I'd prefer this to be set to v0.8.0 AKA c5b7fccd204277076155f10851dad72b76a49317

Comment (SuperQ) [inline vendor/vendor.json]: :+1:

**Comment by beorn7** [comment]
:+1:

**Pull request merged**